themes/bootstrap3/templates/search/blended.phtml

```diff
@@ -0,0 +1,3 @@
+<!-- START of: finna - search/blended.phtml -->
```

Comment:
Do you still need these START/END comments?

Reply:
No, removed!